### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by mraza007**
I have created a separate place for docs . They are hosted at readthedocs
here's the link https://email-dashboard.readthedocs.io/en/latest/
once you guys sign up for the readthedocs i will add you guys as mainters for the docs.
Please let me know if I should add a detailed contribution guide

### CODE REVIEW

**Comment by mraza007** [comment]
Adding the Detailed Contribution guide

File: docs/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2018 Philip Diffenderfer
```
Comment (sadikay) [inline docs/LICENSE.md]: @mraza007 Can you update this line to `Copyright (c) 2018 Email Dashboard`. Just updated in master to this as well.

Comment (mraza007) [reply docs/LICENSE.md]: Sure I will do that

**Comment by mraza007** [comment]
You can merge the pull request

**Pull request merged**